packages/playback-core/src/index.ts

```diff
@@ -12,7 +12,12 @@ export type Metadata = Partial<Options['data']>;
 export type PlaybackEngine = Hls;
 export { mux, Hls, MediaError, Autoplay, UpdateAutoplay, setupAutoplay, addTextTrack, removeTextTrack };
 
+const userAgentStr = globalThis?.navigator?.userAgent ?? '';
+const isAndroid = userAgentStr.toLowerCase().indexOf('android') !== -1;
+
 const MUX_VIDEO_DOMAIN = 'mux.com';
+const HLS_SUPPORTED = Hls.isSupported?.();
```

Comment:
`?.` for SSR support, since hls.js returns an empty object (`{}`) in non-browser envs.